internal/jobservice/application.go

```diff
@@ -30,12 +34,41 @@ func New() *App {
 	return &App{}
 }
 
+// Returns a non-nil error if mis-configuration is unrecoverable.
+func CheckConfig(config *configuration.JobServiceConfiguration) error {
```

Comment:
#testme

Reply:
Addressed in [a465a04](https://github.com/armadaproject/armada/pull/2420/commits/a465a04f4cf23b8f8cebdaabc1c24c657d0a8afd)